view/widget/menu.py

```
@@ -0,0 +1,30 @@
+from view.widget.nbp_api_view_frame import NbpApiView
+from view.model.button import Button
+from view.model.baseframe import BaseFrame
+from view.view_styles import MenuStyle
+from view.widget.nbp_results_view_frame import NbpResultsView
+
+
+class Menu(BaseFrame):
+
+    def __init__(self, view_manager):
```

Ruff

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)